### DISCUSSION TITLE
Inactive Records as a subset of Global Search

### DISCUSSION THREAD

**Discussion by c-forney** [burkybang/SuiteAdvanced-Feedback#127] [Q&A] [answered]
Not sure how possible this is, but I'll shoot my shot. 

I prefer to keep inactive items out of my global search results, but an additional subsection that shows inactive results separated from active results would be helpful and keep me from turning that option on and off in preferences.

**Comment by burkybang** [comment, marked as answer]
Did you know you can dynamically search for inactive records by appending a plus (+) to your search term?

This will find all employees that contain "Adam" including inactive ones:
> emp: Adam+

**Comment by c-forney** [comment]
Well that's a perfect work around. Thank you!
________________________________
